content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,225 @@ weight = 20
 
 +++
 
+## What is Kubeflow?
+
 Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
 Operators can choose what is best for their users, there is no requirement to deploy every component.
-To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
+
+Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
+[Architecture](/docs/started/architecture/) pages.
 
 ### How to install Kubeflow?
 
-There are two pathways to get up and running with Kubeflow, you may either:
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+There are two primary ways to install Kubeflow:
 
-1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
-2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
+1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
+1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
 
 <a id="packaged-distributions"></a>
 <a id="install-a-packaged-kubeflow-distribution"></a>
-## Packaged Distributions of Kubeflow 
+
+## Packaged Distributions of Kubeflow
+
+Packaged distributions are maintained by various organizations and typically aim to provide
+a simplified installation and management experience for Kubeflow.
 
 {{% alert title="Note" color="warning" %}}
-Packaged distributions are developed and supported by their respective maintainers, <b>the Kubeflow community does not endorse or certify any specific distribution</b>.
+Packaged distributions are developed and supported by their respective maintainers.
+The Kubeflow community <strong>does not endorse or certify</strong> any specific distribution.
 
 In the near future, there are plans to introduce <a href="https://github.com/kubeflow/community/blob/master/proposals/kubeflow-conformance-program-proposal.md">conformance testing for distributions</a>, you may track progress on this initiative by following <a href="https://github.com/kubeflow/kubeflow/issues/6485">kubeflow/kubeflow#6485</a>.
 {{% /alert %}}
 
-### Active Distributions
+Some distributions can be deployed on all certified Kubernetes
+distributions <sup>[<a href="https://kubernetes.io/partners/#conformance">1</a>]</sup>, some of them
+can be deployed only in specific Kubernetes environment (e.g. EKS or GKE).
 
-The following table lists <b>active distributions</b> that have <b>had a recent release</b> (within the last 6 months).
+The following table lists distributions which are <em>maintained</em> by their respective maintainers.
 
-<div class="table-responsive">
+<div class="table-responsive distributions-table">
```

Comment:
Based on the discussions in today's community meeting, I think a good compromise is:

1. Keep a single table for all distributions.
1. Re-add the "Distribution Name" column:
      - But leave it empty for all distributions currently called `"Kubeflow on XXX"`.
      - Have each distribution make a PR to update the table, once they decide how they want to brand themselves (within the confines of the Kubeflow trademark rules)
1. Order the columns (from left to right):
     - Platform
     - Kubeflow Version
     - Distribution Name
     - Maintainer
     - Links
1. Sort the rows of the table alphabetically (starting from the left-most column).

__For example:__

Platform | Kubeflow Version |  Distribution Name | Maintainer | Links
--- | --- | --- | --- | ---
All Certified Kubernetes Distributions | 1.8.0<sup>[Release Notes]</sup> | Charmed Kubeflow | Canonical | Website
All Certified Kubernetes Distributions | 1.7.0<sup>[Version Matrix]</sup> | deployKF | Aranui Solutions | Website
Google Kubernetes Engine (GKE) | 1.8.0<sup>[Release Notes]</sup> | - | Google Cloud | Website
IBM Cloud Kubernetes Service (IKS) | 1.8.0<sup>[Release Notes]</sup> | - | IBM Cloud | Website
... | ... | ... | ...

Reply:
I am supportive of this proposal.    One comment - The initial sort might be by Kubeflow Version (rather than by Platform/alpha).  This rewards the distributions that stay up to date.   If the Kubeffow Version is not the 1st column, this ordering might be confusing to the reader.   If the Version is the 1st column, that might be awkward looking.  We might add a comment in the Header - Kubeflow Distributions sorted by latest Kubeflow Version supported.

Reply:
@jbottum from a user perspective, they will be focused on finding a distribution that works on their platform, not finding whatever the "latest" version for other platforms might be.

I think that's why @andreyvelich suggested sorting by the target platform.

Reply:
@thesuperzapper 
>Re-add the "Distribution Name" column:

What if distribution owners don't want to give name for their distribution ?

>Distribution Name order

From my perspective, maintainer column is more important for users than distribution name.

Reply:
> What if distribution owners don't want to give name for their distribution ?

I think we can just put it as `-` in those cases, like when it's not really a distribution (i.e. a ML Platform), but a simple README of how to use the Kubeflow manifests on their platform.

> From my perspective, maintainer column is more important for users than distribution name.

I think most distribution maintainers would prefer to "brand" their distribution as its name, rather than their company name, so my preference is to list "Distribution Name" before "Maintainer",

However, I am open to hearing others' thoughts on this.